contracts/lib/BabyJubJub.sol

```diff
@@ -1,3 +1,4 @@
+// SPDX-License-Identifier: GPL-3.0
```

Comment:
did we do this intentionally?

Reply:
I copy pasted this licence from other files to avoid the Sold compiler warnings. It demands some licence notion in each .sol file. Does it bring any issues?

Reply:
@OBrezhniev check this.